### ISSUE TITLE
Getting current release running is hard and obscure

### ISSUE THREAD

**Issue by dcarosone** [WebGoat/WebGoat#308]
Freshly downloaded as per the instructions from https://s3.amazonaws.com/webgoat-war/webgoat-standalone-7.1-SNAPSHOT-exec.jar

sha1: `f60d1dce0563901097c42c3c7a3abaa62d2ba00e  webgoat-standalone-7.1-SNAPSHOT-exec.jar`

Will not run.  `java -jar webgoat-standalone-7.1-SNAPSHOT-exec.jar` produces a long stream of [exception traces](https://github.com/WebGoat/WebGoat/files/700630/jre-log.txt) then exits.

Ubuntu 16.10, with either Java 8 or 9 (`openjdk-8-jre` or `openjdk-9-jre`)

**Comment by dcarosone** [comment]
Ok, this might actually be 2 issues:

Issue 1: A doc bug / majorly unclear README with respect to "legacy versions", current development, and current release.

I saw the note about 8.x major development, and legacy versions at the top of README.md. The "legacy versions" link goes to 6.x. I wanted 7.1 which was described as the current release.  The quickstart section further down the page has a URL calling itself 7.1, which is what I used. 

I also checked the release downloads for 7.1, but that _does not have_ anything calling itself "standalone".

It turns out that in the wiki, there is an installation page for 7.1 that says to download `webgoat-container-7.1-exec.jar` which is the correct thing.

So I think it would be **hugely useful** to:
* link to the 7.1 installation page at the top of the readme, as or more prominently than the 'legacy' link, so people know where to go for the current release
* change the name of the s3 download link, and generally update the lower part of the readme, to refer to development versions rather than '7.1'

Issue 2. The standalone jar is broken, and misnamed, in the develop branch, because development.
